src/guides/v2.3/extension-dev-guide/build/di-xml-file.md

```diff
@@ -21,7 +21,8 @@ Magento loads the configuration in the following stages:
 
 1. Initial (`app/etc/di.xml`)
 1. Global (`<moduleDir>/etc/di.xml`)
-1. Area-specific (`<moduleDir>/etc/<area>/di.xml`)
+1. Area-specific-frontend (`<moduleDir>/etc/frontend/di.xml`)
+1. Area-specific-backend (`<moduleDir>/etc/adminhtml/di.xml`)
```

Comment:
Hello @rajesh-mage thanks for your contribution. 

These changes are correct but not exhaustive because there can be other areas, like:

- `crontab`
- `webapi_rest`
- `webapi_soap`
- `webapi_soap`

My suggestion is: add all of them or leave the `<area>` placeholder.
What do you think?

Reply:
Hi @aleron75,
Thanks for your review and suggestion. I made some changes to this topic. That is the following,

Magento loads the configuration in the following stages:
1. Initial (app/etc/di.xml)
2. Global (<moduleDir>/etc/di.xml)
3. Area-specific (<moduleDir>/etc/<area>/di.xml)
The Areas are following,
  1. adminhtml
  2. frontend
  3. webapi_rest
  4. webapi_soap
  5. crontab

Kindly review the above and share your suggestions.

Reply:
Hello @rajesh-mage

>  I made some changes to this topic

it's ok for me but I don't see these changes, did you commit and push them?

Reply:
Hi @aleron75,
The changes committed. Kinldy review and confirm it.

Thanks.